vision_agent/agent/vision_agent_coder.py

```diff
@@ -675,6 +675,7 @@ def chat_with_workflow(
         test_multi_plan: bool = True,
         display_visualization: bool = False,
         customized_tool_names: Optional[List[str]] = None,
+        envs: Optional[Dict[str, str]] = None,
```

Comment:
Can we add a docstring for the envs parameter explaining what is this used for ? Since this is a most used function by developers , it might be helpful for them to understand the envs parameter

Reply:
Fixed, thanks for the suggestion!